### DISCUSSION TITLE
Does this have API?

### DISCUSSION THREAD

**Discussion by boringcommerce** [mfts/papermark#395] [General]
Hi. I wanted to use some kind of workflow automation software to auto generate documents using google docs and this. Is it possible to get an API for this?

**Comment by mfts** [comment]
Hey @boringcommerce API is coming

**Reply by boringcommerce** [reply]
Approximately how many weeks/days will it take before its live?

**Reply by opencastlecc** [reply]
@mfts  is the api available yet?
